### PR TITLE
[Fix] Fix some ui bugs

### PR DESCRIPTION
**Pull request by MrQKyber**

### CODE REVIEW

File: KyberNetwork/Resources/Strings.swift [hunk 1]
```diff
@@ -296,4 +296,5 @@ struct Strings {
   static let Staking = "Staking".toBeLocalised()
   static let noRecordFound = "no.record.found".toBeLocalised()
   static let earnIsCurrentlyNotSupportedOnThisChainYet = "earn.is.currently.not.supported.on.this.chain.yet".toBeLocalised()
+  static let emptyTokenDeposit = "No tokens deposited yet.".toBeLocalised()
```
Comment (tungnguyen20) [inline KyberNetwork/Resources/Strings.swift:299]: Please add string to Localizable file

Comment (MrQKyber) [reply KyberNetwork/Resources/Strings.swift]: done

File: KyberNetwork/KyberNetwork/Screens/Tabbars/Earn/ControllersV2/StakingPortfolioViewController.swift [hunk 2]
```diff
@@ -199,19 +208,22 @@ class StakingPortfolioViewController: InAppBrowsingViewController {
...
     viewModel.reloadDataSource()
     portfolioTableView.reloadData()
+    updateUIEmptyView()
   }
   
   override func onAppSelectAllChain() {
     viewModel.chainID = nil
     viewModel.reloadDataSource()
     portfolioTableView.reloadData()
+    updateUIEmptyView()
```
Comment (tungnguyen20) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/Earn/ControllersV2/StakingPortfolioViewController.swift:226]: call `reloadUI` instead

File: KyberNetwork/KyberNetwork/Screens/Tabbars/Earn/ControllersV2/StakingPortfolioViewController.swift [hunk 3]
```diff
@@ -199,19 +208,22 @@ class StakingPortfolioViewController: InAppBrowsingViewController {
...
   }
   
   @objc override func onAppSwitchChain() {
     let currentChain = KNGeneralProvider.shared.currentChain
     viewModel.chainID = currentChain.getChainId()
     viewModel.reloadDataSource()
     portfolioTableView.reloadData()
+    updateUIEmptyView()
```
Comment (tungnguyen20) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/Earn/ControllersV2/StakingPortfolioViewController.swift:219]: call `reloadUI` instead

**Review by tungnguyen20 — APPROVED** [review]

**Pull request merged**